lib/susedistribution.pm

```diff
@@ -239,7 +242,7 @@ sub ensure_installed {
         }
     }
     wait_serial('pkcon-0-', 27) || die "pkcon install did not succeed";
-    send_key("alt-f4");    # close xterm
+    wait_screen_change { send_key("alt-f4"); }    # close xterm
```

Comment:
This is to fix issues like here:
http://pinky.arch.suse.de/tests/838#step/vlc/31
Imho it is easier and safer to fix it here than to ensure that in every testcase using this.
Additionally we avoid unnecessary test failures which we would get when e.g.using assert_screen

Reply:
Sorry, I just fear that this will cause other problems. So please exclude it for now and open a new PR for this with more scenarios covered in local verification

Reply:
Ok, I will seperate this, what problems would you fear?

Reply:
At least wasting time in every single test module which calls this